test/sql/vacuum/test_analyze.test

```diff
@@ -38,12 +38,12 @@ insert into test select range % 5000, range % 5000 from range(10000)
 query T
 select stats(i) from test limit 1
 ----
-[Min: 0, Max: 4999][Has Null: false, Has No Null: true][Approx Unique: 9809]
+[Min: 0, Max: 4999][Has Null: false, Has No Null: true][Approx Unique: 9435]
```

Comment:
Did appx unique count change as a result of the vector size ? Some kind of grouping change for HLL because we are summarizing larger groups ?

Reply:
The approx unique count as is part of the stats does some sampling per vector to avoid significant overhead. Inserting data in larger vectors changes which values are sampled so the result changes somewhat here.